skillmap/src/App.tsx

```diff
@@ -164,9 +168,7 @@ class AppImpl extends React.Component<AppProps, AppState> {
             force: force,
         });
 
-        if (pxt.Util.isLocaleEnabled(useLang!)) {
-            pxt.BrowserUtils.setCookieLang(useLang!);
-        }
+        pxt.BrowserUtils.setCookieLang(useLang!);
```

Comment:
We set he cookie to userLang only if isLocaleEnabled right? Why this change?

Reply:
oh it's being covered by the above case (invalid languages are overwritten by the default), by the time it gets here the useLang should only be an enabled locale! i can add it back though, it doesn't hurt either way

Reply:
But when forcelang is used its not overridden right?

Reply:
OH yes you're totally right, i'll replace the check! hmm @jwunderl do we want to set the user language when the forcelang flag is used? or should we also be skipping that?

Reply:
I would lean towards maintaining the current behavior of not persisting unapproved languages / only keeping them for the session when forcelang is used; forcelang is typically just a test scenario anyway

Reply:
sounds good! i'm moving the setUserLanguage out of `updateLocalizationAsync` and next to the cookie setting, then wrapping them both in this check. this is different from the previous skillmap behavior but i think matches the webapp?